### ISSUE TITLE
Apple Silicon (M1) Incompatibility of hashicorp/random

### ISSUE THREAD

**Issue by LukaGiorgadze** [google/exposure-notifications-verification-server#2211]
### TL;DR
registry.terraform.io/hashicorp/random v2.3.1 does not have a package available for your current
│ platform, darwin_arm64.

**Expected behavior**
`terraform init` success

**Observed behavior**
Failed, incompatible with Apple M1

### Reproduction
`terraform init`

1. I did all the steps from https://github.com/google/exposure-notifications-server/tree/main/terraform
2. When i run `terraform init` i got this error.

**Screenshots**
<img width="892" alt="Screenshot 2021-08-31 at 17 38 06" src="https://user-images.githubusercontent.com/6573695/131512497-ce3d5d9b-8a86-4dc0-981e-a39d996c32b8.png">

**Environment**
Apple Silicon (M1)
- OS: macOS Big Sur
- Version: 11.5.1

**Additional information**
I fixed that issue by upgrading hashicorp/random to 3.1.0 in variables.tf
